events/code_review_events/workflow.py

```diff
@@ -247,14 +283,14 @@ def should_run_test_selection(self, build):
         try:
             if self.should_run_test_selection(build):
                 task = self.community_hooks.triggerHook(
-                    "project-relman", "bugbug-test-select", {"DIFF_ID": build.diff_id},
+                    "project-relman", "bugbug-test-select", {"DIFF_ID": build.diff_id}
```

Comment:
I wonder why this was changed now by pre-commit :S

Reply:
:bomb: